tensorboard/data/provider.py

```diff
@@ -358,6 +372,22 @@ def read_blob(self, ctx=None, *, blob_key):
         """
         pass
 
+    def list_hyperparameters(self, ctx=None, *, experiment_ids):
+        """List hyperparameters metadata.
+
+        Args:
+          ctx: A TensorBoard `RequestContext` value.
+          experiment_ids: A List[string] of IDs of the enclosing experiments.
```

Comment:
Nit: Consider using a more generic annotation, e.g. `Collection` is order is relevant, `Sequence` otherwise (see py tip #44). Here any everywhere.

Reply:
Picked Collection for the experiment_ids argument here.

Picked Collection for the return value of this operation.

Picked Collection for Hyperparameter's domain property wherever I previously specified List.

Now that we require Python 3.8+, it could be worth upgrading this class to pytype. I've added it as a task in our task list.

Reply:
Thanks for adding the task!